source/common/http/header_map_impl.cc

```diff
@@ -589,6 +593,10 @@ HeaderMapImpl::HeaderEntryImpl* HeaderMapImpl::getExistingInline(const char* key
   return nullptr;
 }
 
+HeaderMapImpl::HeaderEntryImpl* HeaderMapImpl::getExistingInline(const char* key) {
```

Comment:
Is this used somewhere? I think they will be implicitly converted to string_view if you eliminate this one.

Reply:
Removed it, and everything builds/tests pass. There were a few spots where this was called with an `std::string` argument that had `.c_str()` called on it. I updated them to use the string directly as argument instead.